atom/browser/api/atom_api_app.cc

```diff
@@ -571,6 +571,8 @@ void App::BuildPrototype(
 #endif
 #if defined(OS_WIN)
       .SetMethod("setUserTasks", base::Bind(&Browser::SetUserTasks, browser))
+      .SetMethod("setCategoryUserTasks",
```

Comment:
So this method sets the category of the given user tasks right?

I wonder if `setUserTasksCategory` or `setCategoryForUsersTasks` might be a better name then?